src/System.Net.Security/src/System/Net/SecureProtocols/_SecuritySafeHandles.Windows.cs

```diff
@@ -219,6 +219,7 @@ public new IntPtr DangerousGetHandle()
         }
 #endif
 
+        #if false //TODO if not used in Nego Stream as well, please remove it.
```

Comment:
Please _do not_ remove code for NegotiateStream. It's on our TODO list and I need to understand where it landed.
Instead, move the code in a Windows-specific implementation.